src/include/cephd/libcephd.h

```diff
@@ -0,0 +1,48 @@
+#ifndef CEPH_LIBCEPHD_H
+#define CEPH_LIBCEPHD_H
```

Comment:
i don't feel strongly about this, but every time we introduce new code into this project. we have a chance to improve it in one way or another. by "improve", we could add more features using the old programming practice, or introduce the improvement with something (not really) new, take C++11 for example, if we really want to be consistent with the existing code, we would be forced to use the C++98 subset. the reason i recommend to use something different in the new code is, taking "#ifdef" as an example, that we could 1) lower the risk of changing the code in a future overhaul to replace the "#ifdef" with "#pragma" 2) the "git blame" would be easier to read after the overhaul.

not to mention that it's not encouraged to change the code only for the cosmetic reasons, IIUC.